### DISCUSSION TITLE
Prod is down! Not able to raise ticket

### DISCUSSION THREAD

**Discussion by workersHive** [vercel/community#4921] [Help]
I deleted my company teams and want to use vercel for only  domain needs! Since we moved hosting to a different server!. 
I assumed domains are separate entities. Now, all the dns and domain are also deleted! 

We are in a situation where we can not access emails, api url. Everything is down. I am not sure how to raise a ticket

**Comment by amyegan** [comment]
Hey @workersHive. If you haven't already, you can [open a ticket](https://vercel.com/help#issues) for the Customer Success Team at [vercel.com/help](https://vercel.com/help). That will give you a more private line of communication with the team.

**Reply by workersHive** [reply]
Thank you. I appreciate it! Yes, I opened the ticket!.  It also seems like I need to set up all the domains and related dns again.
